src/Donations/Components/DonationsForm.tsx

```diff
@@ -53,12 +53,8 @@ function DonationsForm() {
   const router = useRouter();
 
   React.useEffect(() => {
-    if (paymentSetup.minQuantity) {
-      setMinAmt((paymentSetup.minQuantity * paymentSetup.unitCost).toFixed(2));
-    } else {
-      setMinAmt(getMinimumAmountForCurrency(currency));
-    }
-  }, [currency, paymentSetup]);
+    setMinAmt(getMinimumAmountForCurrency(currency));
```

Comment:
This won't work with e.g. TfJ http://localhost:3000/?to=proj_6x3GTD5cMRv0OeQAiIlJZ0Au - please find a solution which works for all kind of projects with every kind of possible configuration (I know our application is quite complex already and the possible configuration dimensions are very large):

<img width="1522" alt="Bildschirmfoto 2022-01-22 um 11 53 02" src="https://user-images.githubusercontent.com/1532418/150635577-b7409401-270f-4cda-8a9e-ec53bdf97e74.png">

Reply:
I must admit this is a bad example as it also won't work now on the develop branch without your change. But this would introduce a bug on production if we release it.